rules/linux/execution_abnormal_process_id_file_created.toml

```
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2022/05/11"
+maturity = "production"
+updated_date = "2022/05/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a Process ID (PID), lock or reboot file created in temporary file storage paradigm (tmpfs) directory
+/var/run. On Linux, the PID files typically hold the process ID to track previous copies running and manage other tasks.
+Certain Linux malware use the /var/run directory for holding data, executables and other tasks, disguising itself or
+these files as legitimate PID files.
+"""
+false_positives = [
+    """
+    False-Positives (FP) can appear if the PID file is legitimate and holding a process ID as intended. To
+    differentiate, if the PID file is an executable or larger than 10 bytes, it should be ruled suspicious.
+    """,
+]
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Abnormal Process ID or Lock File Created"
+references = [
+    "https://www.sandflysecurity.com/blog/linux-file-masquerading-and-malicious-pids-sandfly-1-2-6-update/",
+    "https://twitter.com/GossiTheDog/status/1522964028284411907",
+    "https://exatrack.com/public/Tricephalic_Hellkeeper.pdf"
+]
+risk_score = 43
+rule_id = "cac91072-d165-11ec-a764-f661ea17fbce"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "BPFDoor"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+/* add file size filters when data is available */
+file where user.id == "0" and event.action == "creation" and
```

We should use type instead of action where possible (for consistency). The ordering is just a nit

```suggestion
file where event.type == "creation" and user.id == "0" and
```